.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: go
+
+go:
+  - 1.10
```

Comment:
I think you need to put this in double quotes otherwise travis things this is 1.1